internal/daysteps/daysteps.go

```diff
@@ -10,6 +16,23 @@ var (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+	var duration time.Duration
+	dataArr := strings.Split(data, ",")
+	if len(dataArr) != 2 {
+		return 0, duration, errors.New("input data is not correct")
+	}
+	steps, errAtoi := strconv.Atoi(dataArr[0])
+	if errAtoi != nil {
+		return 0, duration, errors.New("error converting string to number")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```